Cargo.toml

```diff
@@ -22,7 +22,8 @@ is-it-maintained-issue-resolution = { repository = "virtee/sevctl" }
 is-it-maintained-open-issues = { repository = "virtee/sevctl" }
 
 [dependencies]
-sev = { version = "1.1.0", features = ["openssl"] }
+sev = { git="https://github.com/virtee/sev", features = ["openssl"] }
+#sev = { path="../virtee-sev", features = ["openssl"] }
```

Comment:
I will need to update this once `virtee/sev` has a new release, and will remove the commented section (for testing locally).